UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -718,8 +718,20 @@ public void CmdRequestDisarm(GameObject playerToDisarm)
 		}
 	}
 
-	//admin only commands
+	[Command]
+	public void CmdGhostPerformTeleport(Vector3 s3, GameObject localPlayer)
+	{
+		ServerGhostPerformTeleport(s3, localPlayer);
```

Comment:
Pretty unsecure anyone could call cmd from their client and teleport where ever they want. I'd perform a check here to see if this player is actually a ghost before allowing them to teleport

Reply:
Cool

Reply:
if (playerScript.IsGhost)
		{
			playerScript.PlayerSync.SetPosition(s3); //server forces position on player
		}